Paymetheus/Send.xaml

```diff
@@ -232,7 +232,7 @@
                 <ColumnDefinition/>
             </Grid.ColumnDefinitions>
             <TextBlock Grid.Row="0" Style="{StaticResource shellMediumHeaderTextBlockStyle}" Text="Create transaction"/>
-            <TextBlock Grid.Row="1" Style="{StaticResource shellDescriptionTextBlockStyle}" Width="600" Padding="0 10" VerticalAlignment="Top">
+            <TextBlock Grid.Row="1" Style="{StaticResource shellDescriptionTextBlockStyle}" Width="575" Padding="0 10" VerticalAlignment="Top">
```

Comment:
unrelated, and why was this changed? it doesn't even match the width of the description text in the new view.